### PR TITLE
Update Exchange-Teams-interact.md

### PR DESCRIPTION
**Pull request by RuiPereiraTabares**
Based on discussion SEM Yonghong Guo

### CODE REVIEW

**Comment by scanum** [comment]
@JohanFreelancer9 Copy editing is needed for this PR. Thanks.

File: Teams/Exchange-Teams-interact.md [hunk 1]
```diff
@@ -66,7 +66,8 @@ The following table provides a helpful quick reference to feature availability b
 <sup>9</sup> A minimum of an Exchange Online Plan 1 license is also required. For more information, see [Search for Teams chat data for on-premises users](/microsoft-365/compliance/search-cloud-based-mailboxes-for-on-premises-users).
 
 <sup>10</sup> On-premises users can use Teams to update their profile picture even if the `SetPhotoEnabled` Outlook on the Web mailbox policy is set to `false`.
-
+ > [!NOTE]
+ > Setting OOF via the teams client is currently not supported for users that their mailbox is hosted onprem, users should perform this action via the outlook client.
```
Comment (JohanFreelancer9) [inline Teams/Exchange-Teams-interact.md:70]:
```suggestion
 > Setting Out of Office via the Teams client is currently not supported for users whose mailboxes are hosted on-premises; these users should perform this action via the Outlook client.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @RuiPereiraTabares, kindly commit the suggested changes if you agree with them. Many thanks.

**Comment by RuiPereiraTabares** [comment]
@JohanFreelancer9 done

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by scanum** [comment]
@dstrome PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

**Review by joaolloureiro — COMMENTED** [review]
Please prioritize this request

**Comment by RuiPereiraTabares** [comment]
@dstrome Can you please prio the pr 
cc @joaolloureiro

**Comment by dstrome** [comment]
Merged. @RuiPereiraTabares - please join the MicrosoftDocs org so you can be identified as a Microsoft submitter.

**Pull request merged**